vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'vitest/config';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+// eslint-disable-next-line import/no-default-export
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    environment: 'jsdom',
+    globals: true,
```

Comment:
Does this really work? If yes, why do we need to import `vi`?

Reply:
Works in every other project I've used vite in, doesn't work in our current config for some reason though.

Reply:
I think so, tests are passing even though some of the tests didn't have vi imported explicitly.

Personally I prefer to import things directly and leave globals away as much as possible.

But this has implications as mentioned here:

https://vitest.dev/guide/migration.html#globals-as-a-default

Reply:
Oh I am certain it does work generally, I think something is just not right in this PR 😅

Reply:
Oh my bad😂

I don't have a computer with me rn, but can check what is going on tomorrow.